src/dmd/cli.d

```diff
@@ -291,10 +291,6 @@ dmd -cov -unittest myprog.d
             "implement https://github.com/dlang/DIPs/blob/master/DIPs/DIP1000.md",
             "implement $(LINK2 https://github.com/dlang/DIPs/blob/master/DIPs/DIP1000.md, DIP1000 (Scoped Pointers))"
         ),
-        Option("dip1008",
-            "implement https://github.com/dlang/DIPs/blob/master/DIPs/DIP1008.md",
```

Comment:
Btw, this link was a 404. I don't know why there need to be 4 different DIP folders and the DIPs constantly being moved back and forth, it just keeps breaking links everywhere..